pkg/runtime/runtime.go

```diff
@@ -1579,8 +1602,10 @@ func (a *DaprRuntime) publishMessageGRPC(ctx context.Context, msg *pubsubSubscri
 
 	ctx = invokev1.WithCustomGRPCMetadata(ctx, msg.metadata)
 
-	// call appcallback
+	start := time.Now()
 	clientV1 := runtimev1pb.NewAppCallbackClient(a.grpc.AppClient)
+	elapsed := diag.ElapsedSince(start)
+
 	res, err := clientV1.OnTopicEvent(ctx, envelope)
```

Comment:
nits: calculate `elapsed` should behind `OnTopicEvent`.
```
res, err := clientV1.OnTopicEvent(ctx, envelope)
elapsed := diag.ElapsedSince(start)
```

Reply:
Done